lib/hydra/pcdm.rb

```diff
@@ -14,6 +14,7 @@ module PCDM
     autoload :Collection,             'hydra/pcdm/models/collection'
     autoload :Object,                 'hydra/pcdm/models/object'
     autoload :File,                   'hydra/pcdm/models/file'
+    autoload :VersionedFile,          'hydra/pcdm/models/versioned_file'
```

Comment:
So when we update AF:File with a getter/setter for the fedora versionable flag, that will trigger refactors of PCDM:File and then HydraWorks:GenericFile.  The calls to HydraWorks GenericFile and the application levels like Sufia won't change.  Is this correct?  If so, it seems reasonable so long as we remember to make the revisions up the stack in short order.